src/main/java/hudson/plugins/jira/JiraMailAddressResolver.java

```diff
@@ -30,13 +30,14 @@ public String findMailAddressFor(User u) {
 
         String username = u.getId();
 
-        for (JiraSite site : JiraProjectProperty.DESCRIPTOR.getSites()) {
+        for (JiraSite site : JiraGlobalConfiguration.getInstance().getSites()) {
             JiraSession session = site.getSession();
             if (session == null) {
                 continue;
             }
 
-            com.atlassian.jira.rest.client.api.domain.User user = session.service.getUser(username);
+            com.atlassian.jira.rest.client.api.domain.User user = session.service
+                .getUser(username);
```

Comment:
Let's not do partial reformatting as part of a functional PR. In other words, keep the current formatting for unrelated changes, if we want to change line lengths, let's do it for all the source code in a separate PR.